src/reflect/scala/reflect/internal/TreeGen.scala

```diff
@@ -122,7 +122,7 @@ abstract class TreeGen {
   //          val selType = testedBinder.info
   //
   //          // See the test for SI-7214 for motivation for dealias. Later `treeCondStrategy#outerTest`
-  //          // generates an outer test based on `patType.prefix` with automatically dealises.
+  //          // generates an outer test based on `patType.prefix` with automatically dealiases.
```

Comment:
I'll leave this change as-is. `dealiases` appears in `Typers.scala` whereas `unaliases` is a neologism for this code base.